docs/cguru.doc

```diff
@@ -74,10 +74,12 @@
      return value  0: success, 1: success but warning, >1: error (see error.rst)
  
    Notes:
-     * For type 1 and 2, the values in x (and if nonempty, y and z) must
-       lie in the interval [-3pi,3pi). For type 1 they are "sources", but for
-       type 2, "targets". In contrast, for type 3 there are no restrictions on
-       them, or on s, t, u, other than the resulting size of the internal fine
+     * For type 1 and 2, the values in x (and if nonempty, y and z) can be in any
+       interval, they will be folded to [-pi, pi]. Note: for large numbers outside
```

Comment:
there are some places using [-pi, pi], and some using [-pi, pi), maybe stick to [-pi, pi)?